mediorum/ddl/cid_lookup.sql

```diff
@@ -8,22 +8,20 @@ create table if not exists cid_lookup (
 );
 
 -- creator-node creates the Files table, but we need to create it here in case we're running via audius-compose without CNs
-CREATE TABLE IF NOT EXISTS "Files" (
-    "id" SERIAL PRIMARY KEY,
-    "ownerId" INTEGER NOT NULL,
-    "multihash" text NOT NULL,
+CREATE TABLE if not exists "Files" (
+    multihash text NOT NULL,
     "sourceFile" text,
     "storagePath" text NOT NULL,
     "createdAt" timestamp with time zone NOT NULL,
     "updatedAt" timestamp with time zone NOT NULL,
-    "fileUUID" uuid NOT NULL UNIQUE,
+    "fileUUID" uuid NOT NULL,
```

Comment:
fyi I think this unique constraint was added in a CN migration [here](https://github.com/AudiusProject/audius-protocol/blob/main/creator-node/sequelize/migrations/20190711143402-add_uuid_to_users.js#L37). I added the "if not exists" so hopefully it's compatible